lib/resources/majors.tsx

```diff
@@ -5164,6 +5164,201 @@ const bsNeuro: Major = {
   ],
 };
 
+//https://e-catalogue.jhu.edu/engineering/full-time-residential-programs/degree-programs/leadership-education/entrepreneurship-management-minor/#requirementstext
+const minorEntrMgmt: Minor = {
+  degree_name: 'Minor Entrepreneurship and Management',
+  abbrev: 'Minor Entr & Mgmt',
+  department: 'EN Center for Leadership Education',
+  total_degree_credit: 21,
+  url: 'https://e-catalogue.jhu.edu/engineering/full-time-residential-programs/degree-programs/leadership-education/entrepreneurship-management-minor/#requirementstext',
+  wi_credit: 0,
+  distributions: [
+    {
+      name: 'Statistics',
+      required_credits: 3,
+      min_credits_per_course: 3,
+      description:
+        'These courses expose students to the foundations of statistics that are used extensively in business ' +
+        'decision-making. These topics include correlation, estimation, hypothesis testing, linear regression, prediction, ' +
+        'and forecasting. Students may take either two elementary statistics courses or one intermediate/advanced calculus-based' +
+        ' statistics course from the list below.',
+      criteria:
+        'AS.230.205[C]^OR^AS.280.345[C]^OR^AS.280.345[C]^OR^EN.553.111[C]^OR^EN.553.112[C]^OR^EN.661.380[C]^OR^' +
+        'EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]^OR^EN.553.430[C]^OR^EN.560.240[C]',
```

Comment:
where are you getting this list? https://e-catalogue.jhu.edu/engineering/full-time-residential-programs/degree-programs/leadership-education/entrepreneurship-management-minor/#requirementstext:~:text=Credits-,Statistics,-3%2D4